src/etc/inc/util.inc

```diff
@@ -1199,6 +1199,20 @@ function ipcmp($a, $b)
     }
 }
 
+/* compare two IPv6 addresses */
+function ip6cmp($a, $b)
+{
+    $na = inet_pton($a);
```

Comment:
Sorry, meant this line 😉

Reply:
Yes, I thought above it but wasn't brave enough to change ipcmp() at once. Now it seems secure as this function is used only for sorting of dhcp/dhcpv6 static mappings. See a new commit below.

Reply:
let's be brave 👍